DiscordIntegration.cs

```diff
@@ -0,0 +1,211 @@
+using System;
+using System.Collections;
+using RavenM.DiscordGameSDK;
+using Steamworks;
+using UnityEngine;
+
+namespace RavenM
+{
+    public class DiscordIntegration : MonoBehaviour
+    {
+        public static DiscordIntegration instance;
+        
+        public Discord Discord;
+
+        public long discordClientID = 1007054793220571247;
+
+        public long startSessionTime;
+
+        private ActivityManager _activityManager;
+        private void Start()
+        {
+            Environment.SetEnvironmentVariable("DISCORD_INSTANCE_ID", "1007054793220571247");
```

Comment:
(I believe) this is only meant to be used for testing purposes and should represent the Discord client to link to; normally "0" as shown here https://discord.com/developers/docs/game-sdk/sdk-starter-guide#testing-locally-with-two-clients

Using the discordClientID you defined above should work ok.